lib/health_check/suggestion_checker.rb

```diff
@@ -0,0 +1,41 @@
+require "csv"
+require "health_check/calculator"
+require "health_check/suggestion_check"
+
+module HealthCheck
+  class SuggestionChecker
+    def initialize(options = {})
+      @test_data_file = options[:test_data]
+      @search_client = options[:search_client]
+    end
+
+    def run!
+      Logging.logger[self].info("Connecting to #{@search_client.to_s}")
+
+      calculator = Calculator.new
+
+      parsed_checks.each do |search_term, expected_result|
+        suggested_queries = @search_client.search(search_term)[:suggested_queries]
```

Comment:
It would be good to pass `count: 0` to the client to avoid making rummager return a list of results (which should reduce network transfer and load on the server).